CHANGELOG.md

```diff
@@ -20,6 +20,11 @@
   Realm file opened in nonshared mode (via `Group::open()`, as opposed to
   `SharedGroup::open()`).
   PR [#2655](https://github.com/realm/realm-core/pull/2655).
+* Improve ASAN and TSAN build modes (`sh build.sh asan` and `sh build.sh tsan`)
+  such that they do not clobber the files produced during regular builds, and
+  also do not clobber each others files. Also `UNITTEST_THREADS` and
+  `UNITTEST_PROGRESS` options are no longer hard-coded in ASAN and TSAN build
+  modes.
```

Comment:
Could you put in a PR reference after the description? We try to do so for all changes.